doc/build_instructions/windows_msvc.md

```diff
@@ -34,6 +34,8 @@ __NOTE:__ You need to manually make sure and doublecheck if the system you are b
 
  - [CMake](https://cmake.org/download/)
 
+  _Note:_ If you want the easy solution for download other deps, you can download with by executing "install_deps.ps1" in powershell
+
```

Comment:
This should be its own section with instruction on how to use the script.

The script should also be linked and not just referred to.